app/models/version.rb

```diff
@@ -281,6 +281,7 @@ def downloads_count
   def payload
     {
       "authors"                    => authors,
+      "pusher_id"                  => pusher_id,
```

Comment:
do we include raw IDs elsewhere in API responses? Would this make more sense to embed `"pusher" => @pusher&.payload` or something like that?

Reply:
Yes, raw ID is already at profiles API like https://rubygems.org/api/v1/profiles/simi.json and it accepts ID as well https://rubygems.org/api/v1/profiles/42601.json.